editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -487,7 +488,7 @@ protected void prepareDataFormatUIForDataFormat(String dataformat, AbstractCamel
     	CamelModel model = getCamelModel(dataFormatElement);
     	
     	// now create the new fields
-    	DataFormat df = model.getDataformatModel().getDataFormatByName(dataformat);
+    	Eip df = model.getEipModel().getEIPByName(dataformat);

```

Comment:
The catalog provides as choices for the unmarshal type only json or bindy but not the different flavors which are available in dataformat model. The EIP model exactly provides the json and bindy definitions we need. I more and more get the impression that the dataformat model is not really to be used.